diff --git a/node_modules/cipher-base/index.js b/node_modules/cipher-base/index.js
index 176dda9..1150f60 100644
--- a/node_modules/cipher-base/index.js
+++ b/node_modules/cipher-base/index.js
@@ -1,7 +1,7 @@
 'use strict';
 
 var Buffer = require('safe-buffer').Buffer;
-var Transform = require('stream').Transform;
+var Transform = require('readable-stream').Transform;
 var StringDecoder = require('string_decoder').StringDecoder;
 var inherits = require('inherits');
 var toBuffer = require('to-buffer');
